src/styles/GlobalStyle.tsx

```diff
@@ -0,0 +1,37 @@
+/** @jsxImportSource @emotion/react */
+import { css, Global } from '@emotion/react'
+
+const GlobalStyle = () => (
+  <Global
+    styles={css`
+      @import url('https://cdn.jsdelivr.net/gh/orioncactus/pretendard/dist/web/variable/pretendardvariable.css');
```

Comment:
```suggestion
      @import url('https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/vweb/variable/pretendardvariable.css');
```

pretendard 폰트같이 외부에서 가져오는 리소스의 경우 버전이 있기 마련인데 이 경우 버전이 명시된 url을 가져오는 편이 안정적인 프로젝트를 구성하는데 좋습니다 👍